bin/process-csv.js

```diff
@@ -31,143 +31,163 @@
-                            record.StocktakeDate ? Date.parse(record.StocktakeDate) : Date.now()
-                        ))).toISOString(),
-                        updated_at: record.LastUpdate ? (new Date(record.LastUpdate)).toISOString() : Date.now(),
-                        new_from_date: '',
-                        new_to_date: '',
-                        display_product_options_in: 'Block after Info Column',
-                        map_price: '',
-                        msrp_price: record.RRP,
-                        map_enabled: '',
-                        gift_message_available: '',
-                        custom_design: '',
-                        custom_design_from: '',
-                        custom_design_to: '',
-                        custom_layout_update: '',
-                        page_layout: '',
-                        product_options_container: '',
-                        msrp_display_actual_price_type: 'Use config',
-                        // TODO Can we assume this? Or should it be blank?
-                        country_of_manufacture: 'Australia',
-                        // TODO Derive correct value for `quantity_and_stock_status`
-                        additional_attributes: [ 'has_options=0', 'is_returnable=Use config', 'quantity_and_stock_status=In stock', 'required_options=0' ].join(','),
-                        // TODO Do negative numbers in `NoInStock` mean something in particular?
-                        qty: Math.max(0, record.NoInStock),
-                        out_of_stock_qty: 0,
-                        use_config_min_qty: 1,
-                        is_qty_decimal: 0,
-                        // TODO Determine a better value for this?
-                        allow_backorders: 0,
-                        use_config_backorders: 1,
-                        min_cart_qty: 1,
-                        use_config_min_sale_qty: 1,
-                        max_cart_qty: 0,
-                        use_config_max_sale_qty: 1,
-                        is_in_stock: Math.min(1, record.NoInStock),
-                        notify_on_stock_below: '',
-                        use_config_notify_stock_qty: 1,
-                        // TODO Find out what this does
-                        manage_stock: 1,
-                        use_config_manage_stock: 1,
-                        qty_increments: 0,
-                        use_config_qty_increments: 1,
-                        enable_qty_increments: 0,
-                        use_config_enable_qty_inc: 1,
-                        // TODO Find out what this does
-                        is_decimal_divided: 0,
-                        website_id: 1,
-                        deferred_stock_update: 0,
-                        use_config_deferred_stock_update: 1,
-                        // TODO Can we determine related / cross-sell / up-sell products? e.g. By common Author, or subject?
```

Comment:
Maybe, but no.  This data can be added manually after launch, it is not critical and it will be difficult to add.  Have moved RWAHS-403 into "Won't Fix".